modules/react/button/lib/deprecated_Button.tsx

```diff
@@ -38,9 +38,13 @@ const Container = styled('button')<DeprecatedButtonProps & StyledType>(
     position: 'relative',
     cursor: 'pointer',
     outline: 'none',
+
     transition:
       'box-shadow 120ms linear, border 120ms linear, background-color 120ms linear, color 120ms linear',
-    '&:hover:active': {transitionDuration: '40ms'}, // Makes the "down" state of the button happens faster than the hover state, so it animates in correctly.
+
```

Comment:
I wonder what caused these new lines 🤷‍♂️

Reply:
No idea. Old file that finally got touched again with different linting rules?